### PR TITLE
FOGL-8947: Last send Id not updated for single send cycle.

### PR DESCRIPTION
**Pull request by cyberwalk3r**
- Updated last sent id if no readings to send.

### CODE REVIEW

**Review by MarkRiddoch — APPROVED** [review]

**Comment by praveen-garg** [comment]
@gnandan Could you review by test, Please?

File: C/services/north/data_load.cpp [hunk 1]
```diff
@@ -226,6 +229,13 @@ void DataLoad::readBlock(unsigned int blockSize)
 		{
 			// Delete the empty readings set
 			delete readings;
+			n_update_streamId++;
+			if (n_update_streamId > 5) {
```
Comment (gnandan) [inline C/services/north/data_load.cpp:233]: Why if condition is checking for 5, is it some random number?

Comment (cyberwalk3r) [reply C/services/north/data_load.cpp]: This is maximum wait counter when no reading are sent to update streams table. Removed constant.

**Review by gnandan — APPROVED** [review]
Reviewed by test. Stream table is being updated and data is being displayed on GUI as expected.

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**